clusters/build-clusters/01_cluster/README.md

```diff
@@ -29,6 +29,7 @@ Post-install action:
 
 Once we install the cluster, store the installation directory somewhere in case we need to destroy the cluster later on.
 Update password of `kubeadmin` in bitwarden (searching for item called `build_farm_01_cluster `).
+The cert-based kubeconfig file is also uploaded to the same BW item (attachement `b01.kubeconfig`).
```

Comment:
Lets please put `cert` and `admin` in the name of this kubeconfig, we have plenty of kubeconfigs and need to know how this one is special

Reply:
`b01.admin.cert.kubeconfig` and `b02.admin.cert.kubeconfig`.
Reuploaded the items in BW.